packages/design-mui/src/components/MuiEverything/MuiEverything.tsx

```diff
@@ -0,0 +1,87 @@
+import { MuiDataGrid } from "../MuiDataGrid"
+import Grid from "@mui/material/Grid"
+import Tabs from "@mui/material/Tabs"
+import Tab from "@mui/material/Tab"
+import Typography from "@mui/material/Typography"
+import { Eggbert } from "../../assets"
+import type { XEverythingProps } from "@hatchifyjs/react-ui"
+import type { PartialSchema } from "@hatchifyjs/core"
+import type { GetSchemaNames } from "@hatchifyjs/rest-client"
+import { HatchifyEmpty } from "@hatchifyjs/react-ui"
+
+export function MuiEverything<
+  const TSchemas extends Record<string, PartialSchema>,
+  const TSchemaName extends GetSchemaNames<TSchemas>,
+>({
+  children,
+  data,
+  finalSchemas,
+  schemaName,
+  setSelectedSchema,
+  ...props
+}: XEverythingProps<TSchemas, TSchemaName>): JSX.Element {
+  return (
+    <Grid container>
+      <Grid item xs={3} sx={{ backgroundColor: "lightgrey" }} height="auto">
```

Comment:
i don't think we need to match the exact color in mock up - it was Justin hacking something together in a couple of minutes; so feel free to make the style/color/aesthetic for mui-y